packages/site/src/main.css

```diff
@@ -106,4 +106,8 @@ pre.root-pre {
   background: var(--color-surface--background);
   padding: var(--space-large);
   border-radius: var(--radius-base);
+}
+
+section[role="tabpanel"] > div > h1:first-of-type {
```

Comment:
We could probably narrow this down further to be just on the page it'll be used on, but it's good for now!

Reply:
Yeah I can even add a selector that scopes to the tabname="design" if things get weird!